zmon-controller-app/src/main/java/org/zalando/zmon/controller/ZMonRestService.java

```diff
@@ -76,17 +77,16 @@
 
     @RequestMapping(value = "/updateCheckDefinition")
     public ResponseEntity<CheckDefinition> updateCheckDefinition(@RequestBody(required = true) CheckDefinitionImport check) {
-        if (check.getOwningTeam() == null || check.getOwningTeam().equals("")) {
-            if (authService.getTeams().isEmpty()) {
-                check.setOwningTeam("ZMON");
-            } else {
-                check.setOwningTeam((String) authService.getTeams().toArray()[0]);
-            }
+        if (check.getOwningTeam() == null || "".equals(check.getOwningTeam())) {
+            return new ResponseEntity<>(HttpStatus.BAD_REQUEST);
```

Comment:
Is it possible to return a reason for ``BadRequest``? That would be helpful for users to modify the requests.